src/test/ui/sanitizer-use-after-scope.rs

```diff
@@ -0,0 +1,18 @@
+// needs-sanitizer-support
```

Comment:
Please move the test to a relevant sub-directory of `ui/` rather than having it directly in `ui/`.

Reply:
Sanitizer ui tests now have a separate subdirectory.